src/components/MultiSelectGroup/MultiSelectGroup.tsx

```diff
@@ -0,0 +1,103 @@
+import { chakra, Stack } from "@chakra-ui/react";
+import React, { forwardRef } from "react";
+
+import Fieldset from "../Fieldset/Fieldset";
+import { LayoutTypes } from "../../helpers/types";
+import MultiSelect, { MultiSelectWidths } from "../MultiSelect/MultiSelect";
+import useNYPLBreakpoints from "../../hooks/useNYPLBreakpoints";
+
+export interface MultiSelectGroupProps {
+  children: React.ReactNode;
+  /** Additional className to use. */
+  className?: string;
+  id: string;
+  /** Optional prop to overwrite default behavior of the `MultiSelect` component, if nothing gets passed, it is set to `undefined`. */
+  isBlockElement?: boolean;
```

Comment:
As of now, the isElementBlock behavior is so that when the `layout` prop gets set to `"column"`, the isBlockElement will be set to true by default. Having no default value of the `isBlockElement` prop in `MultiSelectGroup`  allows the user to set `isBLockElement=false` even when `layout="column"`. 
However, if that is not an option that should be available to the user, `isBlockElement` could be set to `false` by default (the same default as on the `MultiSelect` component). Then the logic in MultiSelectGroup could be `const finalIsBlockElement = layout === "column" ? true : isBlockElement;`
@nypl-wluisi @bigfishdesign13, how would you like me to proceed?